File: LV1/1.4/numberfun/numberfun.go
package main

import "fmt"

func solvable(a float64, b float64, c float64) bool {
	if a+b == c || a*b == c {
		return true
	} else if a-b == c || b-a == c {
		return true
	} else if a/b == c || b/a == c {
		return true
	}
	return false
}

func main() {
	var n int
	var a, b, c float64
	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		fmt.Scanf("%f %f %f", &a, &b, &c)
		if solvable(a, b, c) {
			fmt.Println("Possible")
		} else {

			fmt.Println("Impossible")
		}
	}
}

File: LV1/1.4/numberfun/numberfun.cpp
#include <cstdio>

bool solvable(double a, double b, double c) {
    if (a + b == c || a * b == c) return true;
    if (a - b == c || b - a == c) return true;
    if (a / b == c || b / a == c) return true;
    return false;
}

int main() {
    int n;
    double a, b, c;
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%lf %lf %lf", &a, &b, &c);
        if (solvable(a, b, c)) {
            puts("Possible");
        } else {
            puts("Impossible");
        }
    }
}